pypeit/core/extract.py

```diff
@@ -1850,19 +1900,35 @@ def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, order_vec, maskslit
     gdorders = np.arange(norders)[np.invert(maskslits)]
     for iord in gdorders: #range(norders):
         msgs.info('Finding objects on order # {:d}'.format(order_vec[iord]))
-        thismask = slitmask == gdslit_spat[iord]
-        inmask_iord = inmask & thismask
+        gpmmask = slitmask == gdslit_spat[iord]
```

Comment:
This is not a gpmmask, so do not change its name. It is a mask indicating what is on this slit.

Reply:
Fixed by me.